src/endpoints/tokenizers.js

```diff
@@ -298,11 +298,13 @@ function createSentencepieceDecodingHandler(tokenizer) {
 
             const ids = request.body.ids || [];
             const instance = await tokenizer?.get();
-            const text = await instance?.decodeIds(ids);
-            return response.send({ text });
+            const ops = ids.map(id => instance.decodeIds([id]));
+            const chunks = await Promise.all(ops);
+            const text = chunks.join('');
+            return response.send({ text, chunks });
```

Comment:
`decodeIds` only gives you back the full joined string for whatever array of IDs you pass into it, so I have to split it into one async call per token, run concurrently.